### PR TITLE
cpu/stressng_cpu.py: adding test for cpu stress

### PR DESCRIPTION
**Pull request by rohan-r-deshpande**
- adding a new file by the name stressng-cpu.py for stressing the CPU
- added yaml file   : stressng_cpu.py.data/stressng_cpu.yaml
- added README file : stressng_cpu.py.data/README

Signed-off-by: Rohan Deshpande <rohan_d@linux.ibm.com>

### CODE REVIEW

File: cpu/stressng_cpu.py.data/README [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+        - longjmp
+        - lsearch
+        - matrix
+        - qsort
+        - str
+        - stream
+        - tsearch
+        - vecmath
+        - wcs
+
```
Comment (PraveenPenguin) [inline cpu/stressng_cpu.py.data/README:32]: please remove this white space

File: cpu/stressng_cpu.py.data/README [hunk 2]
```diff
@@ -0,0 +1,32 @@
+ Description:
```
Comment (PraveenPenguin) [inline cpu/stressng_cpu.py.data/README:1]: remove this indentation

File: cpu/stressng_cpu.py.data/stressng_cpu.yaml [hunk 3]
```diff
@@ -0,0 +1,3 @@
+runtime: 7200 
+url: "https://github.com/ColinIanKing/stress-ng/archive/master.zip"
+crt_stressors: ["bsearch", "context", "cpu", "crypt", "hsearch", "longjmp", "lsearch", "matrix", "qsort", "str", "stream", "tsearch", "vecmath", "wcs"]
```
Comment (PraveenPenguin) [inline cpu/stressng_cpu.py.data/stressng_cpu.yaml:3]: please wrap in 80 Char

**Review by PraveenPenguin — CHANGES_REQUESTED** [review]

File: cpu/stressng_cpu.py.data/README (hunk 1 above)
Comment (rohan-r-deshpande) [reply cpu/stressng_cpu.py.data/README:32]: The reason for the whitespace is - later we get a "No Newline..." remark during the checks. Hence added a newline at the end. Please refer here : https://github.com/avocado-framework-tests/avocado-misc-tests/pull/2444#discussion_r1006830901

Comment (rohan-r-deshpande) [reply cpu/stressng_cpu.py.data/README]: Done.

File: cpu/stressng_cpu.py.data/stressng_cpu.yaml (hunk 3 above)
Comment (rohan-r-deshpande) [reply cpu/stressng_cpu.py.data/stressng_cpu.yaml]: Done

**Review by PraveenPenguin — APPROVED** [review]

**Pull request merged**